recipes-core/glibc/files/0031-Add-ARC-architecture.patch

```diff
@@ -1,18 +1,6 @@
-glibc: Update to arc-2.29-2019.02.08 tag
 
-This was generated from Synopsys GitHub Glibc WIP fork [1] this way:
```

Comment:
Please keep this header in place. This will help you or another person to generate this diff!